issuetracker-api/src/main/java/com/comento/issuetracker/domain/issue/repository/IssueRepository.java

```diff
@@ -4,7 +4,22 @@
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
+
 @Repository
 public interface IssueRepository extends JpaRepository<Issue, Long> {
 
+    //이슈 목록 조회
+    public List<Issue> findByUseYnOrdeByRegAt(String useYn);
+
+
+    // 이슈 제목으로 조회 -> 추후 조건 추가
+    public List<Issue> findByIssueTitleLike(String issueTitle);
```

Comment:
```suggestion
    // 이슈 제목으로 조회 
    // TODO 추후 조건 추가
    public List<Issue> findByIssueTitleLike(String issueTitle);
```

이후에 할 내용을 적어두는 것은 좋은 습관이라고 생각합니다. 👍
`nit` todo 키워드를 조금 더 활용하면 좋다고 생각합니다.